src/core/logic.py

```diff
@@ -919,3 +919,11 @@ def render_nested_setting(
     )
 
     return rendered_string
+
+def restrict_articles_to_editor_assigned(request, articles):
```

Comment:
Might be worth renaming this, it doesn't actually restrict anything and could be used for other things.